src/rgw/rgw_lc.cc

```diff
@@ -772,7 +772,9 @@ class LCOpAction_CurrentExpiration : public LCOpAction {
       r = remove_expired_obj(oc, !oc.bucket_info.versioned());
     }
     if (r < 0) {
-      ldout(oc.cct, 0) << "ERROR: remove_expired_obj " << dendl;
+      ldout(oc.cct, 0) << "ERROR: remove_expired_obj " 
+      << oc.bucket_info.bucket << ":" << o.key 
+      << " (" << cpp_strerror(r) << ")" << dendl;
```

Comment:
cpp_strerror() already return the error string inside parenthesis

Reply:
I will change it
Thank you